src/cf-notification/src/cf-notification.less

```diff
@@ -0,0 +1,44 @@
+/* ==========================================================================
+   Capital Framework
+   Notifications Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// Color variables
+
+@notification-bg:             #f7f8f9; // @gray-10
```

Comment:
these should use the variable as the value, not the hex code. up to you if you want to fix before merge, i'm fine with it coming later.

Reply:
Ah right, I can fix and then will merge.